packages/radix/src/components/ToggleButton.tsx

```diff
@@ -71,27 +102,23 @@ export const ToggleButton = withTheme(
                  * remove the borders and use box-shadows, because we use negative margins to compensate for adjacent
                  * border.
                  */
-                borderColor: 'transparent',
                 '&:hover': {
-                  borderColor: 'transparent',
+                  borderColor: props.size === 0 ? 'transparent' : 'gray500',
```

Comment:
These type of rules where one variant depends on another is worrying me :) 

It's ok when we're hand writing the code like now, but... you know what I mean.